sunpy/visualization/animator/base.py

```diff
@@ -265,14 +265,14 @@ def _make_axes_grid(self):
         pad_size = Size.Fraction(pad, Size.AxesX(self.axes))
         large_pad_size = Size.Fraction(0.1, Size.AxesY(self.axes))
 
+        button_grid = max((7, self.num_buttons))
```

Comment:
Do these changes affect the logic of this function? I have no idea what is supposed to be happening in here.

Reply:
I don't think so, and we do actually have figure tests for this now :rofl: